lms/djangoapps/course_api/views.py

```diff
@@ -89,7 +89,7 @@ class CourseDetailView(DeveloperErrorViewMixin, RetrieveAPIView):
             }
     """
 
-    serializer_class = CourseSerializer
+    serializer_class = CourseDetailSerializer
```

Comment:
Class comments need to reflect the change in the values and the example response.